nautobot/docs/development/getting-started.md

```diff
@@ -180,123 +183,18 @@ Additional useful commands for the development environment:
-
-Create `invoke.yml` as described above, then create `development/docker-compose.override.yml` with the following contents:
-
-```yaml
----
-services:
-  nautobot:
-    env_file:
-      - "override.env"
-```
-
-The `docker-entrypoint.sh` script will run any migrations and then look for specific variables set to create the superuser. The `docker-entrypoint.sh` script is copied in during the Docker image build and will read from the default `dev.env` as the `env_file` until you override it as seen above.
-
- Any variables defined in this file will override the defaults. The `override.env` should look like the following:
-
-```bash
-# Superuser information. NAUTOBOT_CREATE_SUPERUSER defaults to false.
-NAUTOBOT_CREATE_SUPERUSER=true
-NAUTOBOT_SUPERUSER_NAME=admin
-NAUTOBOT_SUPERUSER_EMAIL=admin@example.com
-NAUTOBOT_SUPERUSER_PASSWORD=admin
-NAUTOBOT_SUPERUSER_API_TOKEN=0123456789abcdef0123456789abcdef01234567
-```
-
-The variables defined above within `override.env` will signal the `docker-entrypoint.sh` script to create the superuser with the specified username, email, password, and API token.
-
-After these two files are created, you can use the `invoke` tasks to manage the development containers.
-
-#### Microsoft Visual Studio Code Integration
-
-For users of Microsoft Visual Studio Code, several files are included to ease development and integrate with the [VS Code Remote - Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers). The following related files are found relative to the root of the project:
-
-- `.devcontainers/devcontainer.json` - Dev. container definition
-- `nautobot.code-workspace` - VS Code workspace configuration for Nautobot
-- `development/docker-compose.debug.yml` - Docker Compose file with debug configuration for VS Code
-
-After opening the project directory in VS Code in a supported environment, you will be prompted by VS Code to **Reopen in Container** and **Open Workspace**. Select **Reopen in Container** to build and start the development containers. Once your window is connected to the container, you can open the workspace which enables support for Run/Debug.
-
-To start Nautobot, select **Run Without Debugging** or **Start Debugging** from the Run menu. Once Nautobot has started, you will be prompted to open a browser to connect to Nautobot.
+!!! tip
+    To learn about advanced use cases within the Docker Compose workflow, see the [Docker Compose Advanced Use Cases](docker-compose-advanced-use-cases.md/) page.
 
 !!! note
-    You can run tests with `nautobot-server --config=nautobot/core/tests/nautobot_config.py test nautobot` while inside the Container.
+    If you are making edits to Nautobot's documentation in the Docker Compose workflow or otherwise needing to serve the docs locally, it is necessary to run a Python virtual environment:
+    
+    - `pip3 install poetry`
```

Comment:
My only gripe is that this isn't the "dogmatic" way to install Poetry and might result in some issues being encountered in some cases, depending on the libraries in a given virtualenv. But in all honesty that is somewhat outside of the scope of the intent of this document.

If it's not a huge update, perhaps reference the "Install PoetryZ" section (https://nautobot--870.org.readthedocs.build/en/870/development/getting-started/#install-poetry) vs. explicitly providing the command?

Otherwise, NBD.

Reply:
Love the suggestion.  Done.